R/rules-spacing.R

```diff
@@ -1,8 +1,16 @@
 math_token <- c("'+'", "'-'", "'*'", "'/'", "'^'")
+
+special_token <- lookup_new_special()
```

Comment:
This looks brittle, we'd need an `#' @include` roxygen tag to ensure that the file where `lookup_new_special()` is defined is processed before this code is seen.